Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -456,6 +456,24 @@ public ChaikinMoneyFlow CMF(Symbol symbol, int period, Resolution? resolution =
 
         }
 
+        /// <summary>
+        /// Creates a new Chande's Variable Index Dynamic Average indicator.
+        /// </summary>
+        /// <param name="symbol">The symbol whose VIDYA we want</param>
+        /// <param name="period">The period over which to compute the VIDYA</param>
+        /// <param name="resolution">The resolution</param>
+        /// <param name="selector">Selects a value from the BaseData to send into the indicator, if null defaults to the Value property of BaseData (x => x.Value)</param>
+        /// <returns>The VariableIndexDynamicAverage indicator for the requested symbol over the specified period</returns>
+        [DocumentationAttribute(Indicators)]
+        public VariableIndexDynamicAverage VIDYA(Symbol symbol, int period, Resolution? resolution = null, Func<IBaseData, decimal> selector = null)
```

Comment:
The position of the indicators in this file are alphabetical, will need to moves this helper method to respect this